src/app/views/panels/widgets/manager.tsx

```diff
@@ -1225,6 +1225,10 @@ export class WidgetManager
       </div>
     );
   }
+
```

Comment:
It seems like the other classes are using the `fluentui_manager.tsx` version of the WidgetManager instead of this version. Can `manager.tsx` be re-factored on a new PR to avoid having to duplicate methods between the two versions of WidgetManagers.

Reply:
This is not possible in this case. We have a common interface for `fluentui_manager.tsx` and `manager.tsx`. We have to implement methods for both managers. `manager.tsx` will be removed soon.